tools/ansible/tasks/wsu/main.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- hosts: localhost
+  vars:
+    project_root: "{{ playbook_dir }}/../../../.."
+    kubelet_location: "https://dl.k8s.io/v1.14.0/kubernetes-node-windows-amd64.tar.gz"
+    tmp_path: "/tmp/wsu"
+
+  tasks:
+    - name: Gather required files
+      block:
+
+        - name: Create directory to store files to be used later
+          file:
```

Comment:
Can use tempfile module instead and register the variable

Reply:
There are issues using tempfile with remote copy that I was running into. I'm unable to find the github issue that confirmed my problem, but switching to using /tmp/ fixed it.

Reply:
Got tempfile working in accordance with your instructions!